### PR TITLE
[PBW-5131] More work

### PR DESCRIPTION
**Pull request by mityaha**
Fixed a bug when on mouse down without moving smaill images were staggered on top of central image, fixed unit test problem with padding parameter

### CODE REVIEW

File: js/components/thumbnailCarousel.js [hunk 1]
```diff
@@ -27,19 +27,22 @@ var ThumbnailCarousel = React.createClass({
                    centerThumbnailHeight: carousel.clientHeight});
   },
 
+  componentWillReceiveProps: function(nextProps) {
```
Comment (mistermendez) [inline js/components/thumbnailCarousel.js]: why is this here? It will always set `this.props.update` to false

Comment (mityaha) [inline js/components/thumbnailCarousel.js]: This is a workaround for conflict with test code, I was telling you about it on Friday.  This method is called by real code, but it is not called by test code.  So, after it is set to false, real code will be just comparing properties, i.e. do the right job.

**Comment by mityaha** [comment]
Looks good
